examples/custom/autocomplete/autocomplete_input.py

```diff
@@ -0,0 +1,22 @@
+from bokeh.core.properties import List, String, Int, Bool
+from bokeh.models.widgets import TextInput
+
+class AutocompleteInput(TextInput):
+    """ Single-line input widget with auto-completion. """
+
+    __implementation__ = "autocomplete_input.coffee"
+    __css__ = ["https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.css"]

```

Comment:
The prefixed jquery css does not include ui-autocomplete, so if I remove `__css__` or use the current AutocompleteInput widget, the autocomplete menu is a mess. Maybe it was an oversight, but my takeaway from that and some other open issues was that `jquery-ui` was being dropped from bokeh's packaged css, and shifted to these custom extensions. (The `.addClass("bk-ui-autocomplete")` does not do anything above.)